### DISCUSSION TITLE
adding a pluggin for web develoment

### DISCUSSION THREAD

**Discussion by alavaelian** [LunarVim/LunarVim#3000] [Q&A]
### Feature Description

i want help to install this plugin on lunar vim https://github.com/ray-x/web-tools.nvim it use plug instead of packer so i dont know how to install it or if is some alternative to it, if some one have others usefull plugins for web develoment please tellme

**Comment by rebuilt** [comment]
https://www.lunarvim.org/plugins/#user-plugins

Add the following lines to your config
```lua
lvim.plugins = {
  {"ray-x/web-tools.nvim"}
}
```

Close and restart LunarVim and run `:PackerSync` or `:PackerInstall`.

Follow the rest of the instructions in the README of that plugin to install the dependencies for this plugin

**Reply by Frany-oss** [reply]
My Lunarvim was working fine with the plugins I installed:   
```
lvim.plugins = {
   { 'norcalli/nvim-colorizer.lua' },
   { 'maxmellon/vim-jsx-pretty' },
   { "lunarvim/colorschemes" }
}
```
and then, when I wanted tu Sync my plugins, lunarvim told me if I wanted to delete or remove them. upon insinting on removing them I did so I could install them again.

But now  I can't install them, I try and nothing happens. What could be the issue?

**Comment by Azanul** [comment]
the referenced page has moved from https://www.lunarvim.org/plugins/#user-plugins to https://www.lunarvim.org/docs/configuration/plugins/user-plugins
